src/py/flwr/common/message.py

```diff
@@ -300,17 +300,17 @@ def _create_reply_metadata(self, ttl: float) -> Metadata:
     def create_error_reply(
         self,
         error: Error,
-        ttl: float,
```

Comment:
Do we want to enable the user to set a custom TTL? We could use the same `ttl: float | None = None` approach as in `create_reply`